scss/mixins/_forms.scss

```diff
@@ -72,8 +72,27 @@
         // Boosted mod: no box-shadow
 
         // Boosted mod: focus styles don't match validation state
+
+        // Boosted mod: Add border color to buttons for form element QuantitySelector
+        & ~ button {
```

Comment:
If placed here, it is not limited to the quantity selector

Reply:
Done